components/auth/RegisterSignIn.tsx

```diff
@@ -92,23 +93,27 @@ const SignIn: FC<TProps> = ({ userName, onChangePageState }) => {
 
     if (validate()) {
       setLoading(true);
-
       // call api
-      const userInfo = {
-        name: userName,
-        account,
-        password,
-      };
-      alert(JSON.stringify(userInfo));
-
+      const [err, issuer] = await signIn({
```

Comment:
`const [err, user] = await signIn({ ... })`

please `user`  because this object should mean user ?